sessions/react-custom-hooks/advanced-fetch_solution/README.md

```
@@ -0,0 +1,50 @@
+  const {data: joke, isLoading} = useFetch('...')
+  ```
+- Use the new `isLoading` value to display a `⏳` icon instead of the joke text when `isLoading` equals `true`.
+
+### Caching
+
+Let's implement a simple cache for our useFetch hook. We will use the url as the key to our cache to find the correct cached value.
+
+- Add a third state to our `useFetch` hook called `cache`. Initialize it as an empty object.
+- Inside the `fetchData` function after updating the value state, add a new entry to our cache. Use the current `url` as the key and the fetched data as the value. 
+ 
+Hint: you can use the following syntax to use a variable as the key for an object:
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)